lusid-sdk-python/tests/finbournetest.py

```diff
@@ -431,6 +442,154 @@ def create_property_definition_test(self, property_request, property):
         self.assertEqual(property.data_type_id.scope, property.data_type_id.scope)
         self.assertEqual(property.data_type_id.code, property.data_type_id.code)
 
+    def reconciliation_asserts(self,
```

Comment:
why does this not use `/api/portfolios/$reconcileholdings`?

Reply:
As discussed, these assertions are used to verify that the reconciliation has returned the correct result. Now to check that the reconciliation has returned what we expected we need to work out the difference between our holdings in the internal scope and those in the fund accountant scope and check that this is what the reconciliation returned as breaks. 

This is somewhat challenging as it requires me to compare holdings which I don't have the raw data for. This is because I have the raw data for the initial holdings and I have the raw data for the transactions, but adding the transactions to the initial holdings is done inside LUSID and thus I haven't stored locally the updated holdings which we need to compare.

This is why I have used get_holdings to get the holdings from LUSID and then compared them with the returned reconciliation breaks rather than passing in the raw result that I expect.

I think that this is a better approach then trying to manually add my transactions to my holdings outside of LUSID as we need to check that the reconciliation has worked at multiple different effectiveAt times.

What do you think?

Reply:
We should be able to do something like https://github.com/finbourne/lusid-sdk-csharp/blob/master/lusid-sdk-csharp/LusidSdk.Tests/LusidApiTests.cs#L483, we shouldn't be testing that the functionality itself works as that is tested inside LUSID

Reply:
As per our discussion abstracted away